v63002/u_inref/gtm8644.csh

```diff
@@ -0,0 +1,18 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+setenv SHELL /usr/local/bin/tcsh
+$ydb_dist/mumps -run shellfn^gtm8644
+$ydb_dist/mumps -run quotesfn^gtm8644
```

Comment:
I would additionally like a test of the fact that only $SHELL shows up instead of /bin/sh like the release note describes. Towards that, ps --forest -ef will give a good listing of the process tree while inside a ZSYSTEM and we expect to see only $SHELL there, no "sh".